Dockerfile.mod5

```diff
@@ -0,0 +1,49 @@
+FROM python:3.9-slim-buster AS base
```

Comment:
Why have you switched this into a separate Dockerfile? This has now broken your production & dev containers (at least when using docker-compose & following the instructions in your Readme)

Instead we should be able to have these all cleanly in a single Dockerfile. It looks like the changes you actually made to your testing Dockerfile are minimal so this shouldn't be an issue

Reply:
I was not clear on project requirement. Now I switched to single dockerfile.